docs/manifest.json

```diff
@@ -522,12 +522,12 @@
         { "title": "User" },
         [
           ["User", "/references/javascript/user/user"],
```

Comment:
What do you think of calling this "User object" in the sidebar, to at least make it a little less weird to open the "User" dropdown and just see "User" again?

Reply:
great minds think alike ! I was thinking this for all the classes - I agree and I'll update accordingly